src/pip/_internal/wheel.py

```diff
@@ -725,6 +725,56 @@ def _contains_egg_info(
     return bool(_egg_info_re.search(s))
 
 
+def compute_ephem_cache(
```

Comment:
Why do I always nit-pick?

```suggestion
def should_use_ephemeral_cache(
```